<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FolderMatchedAgainstFileWidget</class>
 <widget class="QWidget" name="FolderMatchedAgainstFileWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>124</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>7</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>11</number>
   </property>
   <property name="bottomMargin">
    <number>11</number>
   </property>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="chooseContainer" native="true">
     <layout class="QVBoxLayout" name="chooseLayout">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="CloudStalledIssueChooseWidget" name="remoteCopy" native="true"/>
      </item>
      <item>
       <widget class="LocalStalledIssueChooseWidget" name="localCopy" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CloudStalledIssueChooseWidget</class>
   <extends>QWidget</extends>
   <header>stalled_issues_cases/LocalAndRemoteChooseWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LocalStalledIssueChooseWidget</class>
   <extends>QWidget</extends>
   <header>stalled_issues_cases/LocalAndRemoteChooseWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
